stdlib/@tests/test_cases/check_importlib_resources.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+import importlib.resources
+import pathlib
+import sys
+
+
+class _CustomPathLike:
+    def __init__(self, path: str) -> None:
+        self.path = path
+
+    def __fspath__(self) -> str:
+        return self.path
```

Comment:
Since this code is never executed, it can be simplified to:

```suggestion
class _CustomPathLike:
    def __fspath__(self) -> str:
        return ""
```